cfn-resources/org-invitation/cmd/resource/resource.go

```diff
@@ -57,26 +57,28 @@ func Create(req handler.Request, prevModel *Model, currentModel *Model) (handler
 		currentModel.Profile = aws.String(profile.DefaultProfile)
 	}
 
-	client, peErr := util.NewMongoDBClient(req, currentModel.Profile)
+	client, peErr := util.NewAtlasClient(&req, currentModel.Profile)
+	atlasV2 := client.AtlasV2
+
 	if peErr != nil {
 		return *peErr, nil
 	}
 
-	invitationReq := &mongodbatlas.Invitation{
-		TeamIDs:  currentModel.TeamIds,
+	invitationReq := &admin.OrganizationInvitationRequest{
+		TeamIds:  currentModel.TeamIds,
 		Roles:    currentModel.Roles,
-		Username: *currentModel.Username,
+		Username: currentModel.Username,
 	}
-	invitation, res, err := client.Organizations.InviteUser(context.Background(), *currentModel.OrgId, invitationReq)
+	invitation, res, err := atlasV2.OrganizationsApi.CreateOrganizationInvitation(context.Background(), *currentModel.OrgId, invitationReq).Execute()
 	if err != nil {
 		_, _ = log.Warnf("Create - error: %+v", err)
-		return progressevents.GetFailedEventByResponse(err.Error(), res.Response), nil
+		return progressevents.GetFailedEventByResponse(err.Error(), res), nil
 	}
-	currentModel.Id = &invitation.ID
+	currentModel.Id = invitation.Id
 
 	if err != nil {
 		_, _ = log.Warnf("Read - error: %+v", err)
```

Comment:
i'm favoring to remove these logs when error as you'll get the error so it's kind of duplicating error info

Reply:
Just to be confirm, are errors returned from the response logged by default by cfn? In that case I agree we can remove

Reply:
correct, FAILED and SUCCESS events are logged automatically, e.g. in Create database-user the code is this, no log: 		return progressevent.GetFailedEventByResponse(err.Error(), resp), nil

and the Cloudwatch logs for a failed example: 

![failed](https://github.com/mongodb/mongodbatlas-cloudformation-resources/assets/430982/fbef9b86-3917-41a2-9011-0a5985bcca05)

Reply:
> Just to be confirm, are errors returned from the response logged by default by cfn? In that case I agree we can remove

To follow up here: we might not want to remove them entirely because they can be helpful when running CFN tests locally or during the release process to add extra information for debugging purposes. 

In this case, the log is quite useless (I agree we should remove it) but there may be cases where having an extra log may be useful while investigating issues.